### ISSUE TITLE
PHOENIX atmospheres

### ISSUE THREAD

**Issue by DanielAndreasen** [DanielAndreasen/MOOGme#40]
How about some PHOENIX atmospheres too. They seem to be the future anyway.

**Comment by MariaTsantaki** [comment]
Do you know which group works on that?

**Comment by DanielAndreasen** [comment]
Göettingen: http://phoenix.astro.physik.uni-goettingen.de/

**Comment by MariaTsantaki** [comment]
They provide atmospheres in fits format.
ftp://phoenix.astro.physik.uni-goettingen.de/AtmosFITS/

> Atmospheres

Posted on August 6, 2013	

We’ve now also published the atmospheric structures for all our models. They are available for download as FITS binary tables with the columns tau, temp, pgas and rho.

> 

I think if we convert to kurucz format, this is all we need. :)

**Comment by DanielAndreasen** [comment]
Yes, maybe. I wonder how MOOG works. Does it just read the atmosphere differently according to the keyword (KURUCZ, MARCS, etc.), or does it also apply different physics? If the physics is the same, then let's just convert to Kurucz and we should be good to go. I can take a look at the source code tomorrow

**Comment by DanielAndreasen** [comment]
Okay, I've taken a look at the source code (`Inmodel.f` specifically), and the same physics is applied to all the atmosphere models. Since the columns are the same for all, MOOG compute the columns needed to proceed.
That means if we can make PHOENIX models look like Kurucz, we are good to go. I guess it will take some work, but we only have to create the grid once.

To proceed:
 - [ ] Check which columns PHOENIX has compared to Kurucz
 - [ ] Find a way to calculate the other columns needed
 - [ ] Save the PHOENIX models in a grid similar to Kurucz

**Comment by MariaTsantaki** [comment]
MOOG requires: rho, T, Pgas, Ne, kappaross.
PHOENIX have: rho, T, Pgas, tau
I think we have to ask them for the quantities missing..

**Comment by DanielAndreasen** [comment]
I think it is possible to make it look like the Marcs model atmospheres, then we can set the `MARCS` (or something like that) in the header of the interpolated atmosphere.
